<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleSettingsWidget</class>
 <widget class="QWidget" name="StyleSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="text_editor">
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Text</string>
     </property>
     <widget class="QLabel" name="fontLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Font:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QFontComboBox" name="fontBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>112</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sizeBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="text_color">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>32</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="halign">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Horizontal Alignment:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="bold">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Bold</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="italicize">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Italicize</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="underline">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Underline</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="valign">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertical Alignment:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>36</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Margin:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="margin">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>fontLabel</zorder>
     <zorder>fontBox</zorder>
     <zorder>sizeBox</zorder>
     <zorder>text_color</zorder>
     <zorder>halign</zorder>
     <zorder>label_3</zorder>
     <zorder>bold</zorder>
     <zorder>italicize</zorder>
     <zorder>underline</zorder>
     <zorder>label</zorder>
     <zorder>valign</zorder>
     <zorder>label_5</zorder>
     <zorder>label_8</zorder>
     <zorder>margin</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="frame_editor">
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Container</string>
     </property>
     <widget class="QPushButton" name="bg_color">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Background Color:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Border Color:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="border_color">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Border Width:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="border_width">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
     <widget class="QPushButton" name="border_clear">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bg_clear">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="preview_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
